cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -241,8 +255,7 @@ public KubeTwillPreparer withStatefulRunnable(String runnableName, boolean order
     return this;
   }
 
-  @Override
-  public SecureTwillPreparer withIdentity(String runnableName, String identity) {
```

Comment:
why make this change?

Reply:
Dennis recommended to fold identity into SecurityContext also. 

Do you see a downside with that?

Reply:
I see. I think that should be fine. I asked because it was not part of the impl before.
Also if its only being called in `KubeTwillPreparer`, ok to make it private?

Reply:
Makes sense. Done!